### PR TITLE
chore: polkadot-v1.11.0

### PR DESCRIPTION
**Pull request by al3mart**
Bump dependencies to follow polkadot v1.11.0 crate versions.

Add configuration file for benchmarks.
Remove try-runtime as a subcommand in favor of its standalone version.

### CODE REVIEW

File: node/src/service.rs [hunk 1]
```diff
@@ -139,11 +139,107 @@ pub fn new_partial(config: &Configuration) -> Result<Service, sc_service::Error>
...
+    task_manager
+        .spawn_essential_handle()
+        .spawn("aura", None, fut);
+
+    Ok(())
+}
+
 /// Start a node with the given parachain `Configuration` and relay chain `Configuration`.
 ///
 /// This is the actual implementation that is abstract over the executor and the runtime api.
```
Comment (Daanvdplas) [inline node/src/service.rs:240]: This line and the empty line above was removed

File: node/src/service.rs [hunk 2]
```diff
@@ -139,11 +139,107 @@ pub fn new_partial(config: &Configuration) -> Result<Service, sc_service::Error>
...
+        move |_, _| async move {
+            let timestamp = sp_timestamp::InherentDataProvider::from_system_time();
+            Ok(timestamp)
+        },
+        &task_manager.spawn_essential_handle(),
+        config.prometheus_registry(),
+        telemetry,
+    )
+}
+
```
Comment (Daanvdplas) [inline node/src/service.rs:168]:
```suggestion
#[allow(clippy::too_many_arguments)]
```

File: runtime/src/benchmarks.rs [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    [frame_system, SystemBench::<Runtime>]
+    [cumulus_pallet_parachain_system, ParachainSystem]
+    [pallet_timestamp, Timestamp]
+    [pallet_balances, Balances]
+    [pallet_sudo, Sudo]
+    [pallet_collator_selection, CollatorSelection]
+    [pallet_session, SessionBench::<Runtime>]
+    [cumulus_pallet_xcmp_queue, XcmpQueue]
+    [pallet_message_queue, MessageQueue]
+);
```
Comment (Daanvdplas) [inline runtime/src/benchmarks.rs:37]: These should be inline with the pallets in the runtime. These will vary for the other templates

File: runtime/src/lib.rs [hunk 4]
```diff
@@ -288,23 +288,10 @@ construct_runtime!(
...
-    );
-}
-
 cumulus_pallet_parachain_system::register_validate_block! {
     Runtime = Runtime,
     BlockExecutor = cumulus_pallet_aura_ext::BlockExecutor::<Runtime, Executive>,
 }
+
+#[cfg(feature = "runtime-benchmarks")]
+mod benchmarks;
```
Comment (Daanvdplas) [inline runtime/src/lib.rs:297]: Prefer to have this at the top if possible?

**Review by Daanvdplas — APPROVED** [review]
Well done!

File: runtime/src/benchmarks.rs (hunk 3 above)
Comment (al3mart) [reply runtime/src/benchmarks.rs:37]: > These should be inline with the pallets in the runtime

What do you mean, have them in the same order than `construct_runtime!` or include all pallets in `construct_runtime!` ?

Comment (Daanvdplas) [reply runtime/src/benchmarks.rs:37]: Both, but it is already the case. I forgot they are overcopied from `runtime/src/lib.rs` so ignore the comment :)

**Review by evilrobot-01 — CHANGES_REQUESTED** [review]
Found various things missing which have been proposed via https://github.com/r0gue-io/base-parachain/pull/27

**Review by AlexD10S — APPROVED** [review]

**Pull request merged**